test/controllers/ChargeSummaryControllerSpec.scala

```diff
@@ -183,6 +177,28 @@ class ChargeSummaryControllerSpec extends ChargeSummaryControllerHelper {
                 document.getElementById("interest-on-your-charge-table").getAllElements.size().equals(0) shouldBe false
                 document.getElementById("payment-history-table").getElementsByTag("caption").text() shouldBe "Balancing payment history"
               }
+              "provided with an id associated to a Late Submission Penalty" in new Setup(testValidFinancialDetailsModelWithLateSubmissionPenalty) {
+                enable(ReviewAndReconcilePoa)
```

Comment:
if we can create an overload version of enable method where we can pass variable list of arguments